DEPLOY.md

```diff
@@ -0,0 +1,72 @@
+<!--
+ This document is formatted one-sentence-per-line, breaking very long sentences at phrase boundaries.
+ This format makes diffs clean and review comments easy to target.
+-->
+
+# Releasing and Deploying go-filecoin
+
+Building, releasing and deploying go-filecoin binaries is handled entirely by CI (CircleCI).
+
+There are two primary release and deploy paths, `User` and `Nightly`.
+
+Linux and MacOS binaries are built and distributed in a Github release for each. For Nightly, the release is marked as a pre-release.
+Additionally, a Docker image is published to ECR containing fixtures and rust-fil-proofs groth parameters.
+
+The binary version is passed as a parameter to trigger the `go-filecoin-infra` [Nightly Devnet deploy](https://github.com/filecoin-project/go-filecoin-infra/blob/filecoin-nightly/.circleci/config.yml) or [Test Devnet deploy](https://github.com/filecoin-project/go-filecoin-infra/blob/filecoin-testnet/.circleci/config.yml).
+@TODO replace 'Test Devnet' with 'User Devnet' after testing
+
+Badges can be found in go-filecoin README.md listing current releases.
+
+## User
+
+The User release channel is intended as a quasi-stable (no SLA or guarantees at this time) release and testing bed
+for Filecoin Developers, Infrastructure Engineers and Testers.
+
+A release and deploy to the User Devnet is triggered by pushing a git tag meeting the following regex patterns
+- `/^\d+\.\d+\.\d+$/`
+- `/^testnet\-\d+\.\d+\.\d+$/ # @TODO remove this after deploy testing complete`
+
+### Deploy instructions
+
+1. Announce a code freeze
```

Comment:
nit: I don't know what a code freeze will do, as we are not branching from master.

Reply:
remove this completely